pdns/dnsdist.hh

```diff
@@ -50,6 +50,7 @@
 #include "uuid-utils.hh"
 #include "proxy-protocol.hh"
 #include "stat_t.hh"
+#include "xsk.hh"
```

Comment:
It might make sense not to include this file in dnsdist.hh, and to instead do a forward declaration of class XskSocket; before the `ClientState` definition.